pkg/systemd/quadlet/quadlet.go

```diff
@@ -969,7 +969,7 @@ func addNetworks(quadletUnitFile *parser.UnitFile, groupName string, serviceUnit
 }
 
 func getAbsolutePath(quadletUnitFile *parser.UnitFile, filePath string) (string, error) {
-	if !filepath.IsAbs(filePath) {
+	if filePath[0] != '%' && !filepath.IsAbs(filePath) {
```

Comment:
Absolutely non-blocking:  A comment on the `%` special casing may help future readers.

But don't bother repushing for it.

Reply:
Following other comments, I changes the implementation to use a method and added an explanation on top of it. PTAL